backend/src/authentication/gamma.strategy.ts

```diff
@@ -0,0 +1,63 @@
+import Strategy, { Authority } from "./strategy.js";
+import passport from "passport";
+import { GammaUser } from "../models/gammaUser.js";
+
+const default_options = {
+  authorizationURL: "http://localhost:8081/api/oauth/authorize",
+  tokenURL: "http://localhost:8081/api/oauth/token",
+  profileURL: "http://localhost:8081/api/users/me",
+  clientID: "id",
+  clientSecret: "secret",
+  callbackURL: "http://localhost:3001/auth/account/callback",
+};
+
+const isAdmin = (authorities: Authority[], groups: String[]): boolean => {
+  if (process.env.MOCK && groups.includes("superadmin")) {
+    return true;
+  }
+  for (const i in authorities) {
+    if (authorities[i].authority == process.env.ADMIN_AUTHORITY || groups.includes("drawit")) {
```

Comment:
Is DrawIT always going to be admin?

Reply:
I agree. This was in the code "borrowed" from bookIT. As we now have decided to have a site admin page this "admin" check will only apply to that site admin page in the future. It has been fixed/changed in the feature/orgs branch. So in this PR it does not do anything. 
My question is if we want to port over the change that is currency in the feature/orgs branch to this PR or if we ignore this in the current PR as it will be fixed in the next?

Reply:
Then I think it's better to keep it in the orgs branch and remove it from this one.

Reply:
Removed the drawIT check, but kept the other stuff as that will stay pretty much the same, and removing it is a lot of work for something that will just come back in a later PR